tests/improver-weighted-blending/11-cycletime.bats

```diff
@@ -31,12 +31,12 @@
 
 . $IMPROVER_DIR/tests/lib/utils
 
-@test "weighted-blending linear coordinate input output" {
+@test "weighted-blending cycletime" {
   improver_check_skip_acceptance
   KGO="weighted_blending/cycletime/kgo.nc"
 
   # Run weighted blending with linear weights and check it passes.
-  run improver weighted-blending 'linear' 'forecast_reference_time' 'weighted_mean' --cycletime '20171129T0900Z'\
+  run improver weighted-blending 'forecast_reference_time' 'weighted_mean' --cycletime '20171129T0900Z'\
```

Comment:
This test gives a much greater weight to the forecast with the earlier cycle time, as you are blending over cycle time, and sorting the coordinate you are blending over in ascending order. I'm not sure this behaviour is clear in the help for the CLI help. I can imagine that if we were doing cycle blending with unequal weights we would want to weight the more recent cycle, and at the moment it does the opposite. A bit of extra information in the help somewhere to explain this might be useful.

Reply:
This now may be covered by IMPRO-949